### ISSUE TITLE
TaskMap construction when loading pyexotica

### ISSUE THREAD

**Issue by christian-rauch** [ipab-slmc/exotica#330]
It seems that the constructor of task maps (e.g. `Point2Line`, `CoM`) gets already called when loading the python module via: `import pyexotica as exo`.
Is this expected? I am trying to setup a ROS publisher in the constructor (for debugging). This fails because they are advertised before I can call `exo.Setup.initRos("node_name", False)` in my python code.
How am I supposed to initialise ROS before loading `pyexotica`?

**Comment by VladimirIvan** [comment]
Yes. Python wrapper does indeed create an instance of the class when the python module gets loaded. This is required to collect all the classes declared in the plugins. Because of this, the constructor should stay empty so that it doesn't create much overhead. `Instantiate` doesn't get called at this time so you can put all your code (ROS related or otherwise) there.

The correct order to initialize ROS is:
```
import pyexotica as exo
exo.Setup.initRos()
...
```

Also, use ```Server::advertise``` instead of the native ROS version to create publishers (subscribers and services have similar calls declared). These calls checks if `exo.Setup.initRos()` has been called. The CoM task is using this.
